Scheduler: add the 'allDayPanelMode' property description (#3814)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxScheduler/1 Configuration/allDayPanelMode.md

---
---
##### shortDescription

> Specifies a display mode for the **All day** panel.

Reviewer: ```suggestion
Specifies the display mode for the **All day** panel.
```

---
#include common-demobutton with {
    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Scheduler/AllDayPanelMode/"
}

![allDayPanelMode set to 'all'](/images/Scheduler/scheduler-all-day-panel-mode.png)

To hide the **All day** panel, set this property to `hidden`. 

> When you set this property to `allDay`, the **All day** panel displays only the appointments that have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true**.

Reviewer 2: ```suggestion
If you set the `allDayPanelMode` property to `allDay`, the **All day** panel displays only the appointments whose [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property is set to `true`.
```

> To additionally display the appointments that have the length equal to or more than 24 hours, assign `all` to this property.

Reviewer 2: ```suggestion
To display the appointments whose length is equal to or exceeds 24 hours in addition to all day events, assign `all` to this property.
```

> To additionally display the appointments that have the length equal to or more than 24 hours, assign `all` to this property.

Reviewer 2: ```suggestion
To display the appointments whose length is equal to or exceeds 24 hours in addition to all day events, assign `all` to this property.
```

> To additionally display the appointments that have the length equal to or more than 24 hours, assign `all` to this property.

Reviewer: ```suggestion
To also display appointments that have a duration equal to or more than 24 hours, assign `all` to this property.
```
